Ginger/GingerCoreNET/RunLib/DynamicExecutionLib/DynamicExecutionManager.cs

```diff
@@ -563,7 +563,8 @@ private static GingerExecConfig GetGingerExecConfigurationObject(Solution soluti
                 AutoFixAnalyzerIssue = runsetExecutor.RunSetConfig.SelfHealingConfiguration.AutoFixAnalyzerIssue,
                 ReprioritizePOMLocators = runsetExecutor.RunSetConfig.SelfHealingConfiguration.ReprioritizePOMLocators,
                 AutoUpdateApplicationModel = runsetExecutor.RunSetConfig.SelfHealingConfiguration.AutoUpdateApplicationModel,
-                SaveChangesInSourceControl = runsetExecutor.RunSetConfig.SelfHealingConfiguration.SaveChangesInSourceControl
+                SaveChangesInSourceControl = runsetExecutor.RunSetConfig.SelfHealingConfiguration.SaveChangesInSourceControl,
+                AutoExecuteInSimulateionMode = runsetExecutor.RunSetConfig.SelfHealingConfiguration.AutoExecuteInSimulateionMode
```

Comment:
Please correct the simulation spelling

Reply:
Done from Ginger side, need to update nuget as well